tests/introspection/tests.py

```diff
@@ -217,6 +217,21 @@ def test_get_constraints_indexes_orders(self):
                 indexes_verified += 1
         self.assertEqual(indexes_verified, 3)
 
+    @skipUnlessDBFeature('can_introspect_foreign_keys')
```

Comment:
I'm not entirely sure if this test case should be run for all backends or just SQLite. On one hand this is an sqlite-specific problem, but other backends should also handle this case, at least in theory.